docs/quickstart/quickstart_chains/cosmos-osmosis.md

```diff
@@ -0,0 +1,384 @@
+# Osmosis Quick Start
+
+[Osmosis](https://osmosis.zone/) is a DEX built on the Cosmos ecosystem. It is designed to allow users to trade tokens from different blockchains that are part of the Cosmos ecosystem. Osmosis uses the IBC protocol to enable the transfer of assets between different blockchains, including [Cosmos Hub](https://github.com/subquery/cosmos-subql-starter/tree/main/CosmosHub/cosmoshub-starter), [Akash](https://github.com/subquery/cosmos-subql-starter/tree/main/Akash/akash-starter), and others.
+
+## Goals
+
+This guide acts as your entrance to a detailed tutorial for configuring a SubQuery indexer that is specifically designed to index swaps occurring on Osmosis. Upon completing this guide, you will possess a solid understanding of the process for indexing data related to a complex DEX such as Osmosis.
+
```

Comment:
@bgodlin We could potentially add here a video from my workshop too: https://www.youtube.com/watch?v=Rp4d4NbVzo4